<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameCube File Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="GCMTab" name="gcm_tab">
       <attribute name="title">
        <string>GCM ISOs</string>
       </attribute>
      </widget>
      <widget class="RARCTab" name="rarc_tab">
       <attribute name="title">
        <string>RARC Archives</string>
       </attribute>
      </widget>
      <widget class="BTITab" name="bti_tab">
       <attribute name="title">
        <string>BTI Images</string>
       </attribute>
      </widget>
      <widget class="J3DTab" name="j3d_tab">
       <attribute name="title">
        <string>J3D Files</string>
       </attribute>
      </widget>
      <widget class="JPCTab" name="jpc_tab">
       <attribute name="title">
        <string>JPC Particle Archives</string>
       </attribute>
      </widget>
      <widget class="BMGTab" name="bmg_tab">
       <attribute name="title">
        <string>BMG Messages</string>
       </attribute>
      </widget>
      <widget class="DOLTab" name="dol_tab">
       <attribute name="title">
        <string>DOL Executables</string>
       </attribute>
      </widget>
      <widget class="Yaz0Yay0Tab" name="yaz0_yay0_tab">
       <attribute name="title">
        <string>Yaz0/Yay0 Compression</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GCMTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.gcm_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RARCTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.rarc_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BTITab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.bti_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>J3DTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.j3d_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JPCTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.jpc_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DOLTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.dol_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Yaz0Yay0Tab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.yaz0_yay0_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BMGTab</class>
   <extends>QWidget</extends>
   <header>gcft_ui.bmg_tab</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
